--- a/Makefile
+++ b/Makefile
@@ -1,11 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vlmcsd
-PKG_VERSION:=svn1113
+PKG_VERSION:=1113
 PKG_RELEASE:=1
 
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-svn$(PKG_VERSION)
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/Wind4/vlmcsd/tar.gz/$(PKG_VERSION)?
+PKG_SOURCE_URL:=https://codeload.github.com/Wind4/vlmcsd/tar.gz/svn$(PKG_VERSION)?
 PKG_HASH:=62f55c48f5de1249c2348ab6b96dabbe7e38899230954b0c8774efb01d9c42cc
 
 PKG_MAINTAINER:=fuyumi <280604399@qq.com>
@@ -26,7 +27,7 @@ define Package/vlmcsd/description
 endef
 
 define Build/Prepare
 	$(call Build/Prepare/Default)
-	rm $(PKG_BUILD_DIR)/GNUmakefile
+	rm -f $(PKG_BUILD_DIR)/GNUmakefile
 endef
 
 define Package/vlmcsd/install
